File: app/src/main/java/wxm/KeepAccount/ui/setting/page/TFSettingRemind.java
package wxm.KeepAccount.ui.setting.page;


/**
 * remind setting page
 * Created by WangXM on 2016/10/10.
 */
public class TFSettingRemind extends TFSettingBase {
    @Override
    public void updateSetting() {
        if (mBSettingDirty) {
            mBSettingDirty = false;
        }
    }

    @Override
    protected int getLayoutID() {
        return 0;
    }

    @Override
    protected boolean isUseEventBus() {
        return false;
    }
}

File: app/src/main/java/wxm/KeepAccount/ui/setting/TFSettingRemind.kt
package wxm.KeepAccount.ui.setting


/**
 * remind setting page
 * Created by WangXM on 2016/10/10.
 */
class TFSettingRemind : TFSettingBase() {
    override fun updateSetting() {
        if (isSettingDirty) {
            isSettingDirty = false
        }
    }

    override fun getLayoutID(): Int {
        return 0
    }

    override fun isUseEventBus(): Boolean {
        return false
    }
}
